### PR TITLE
[Related #76] Race track background

### PR DESCRIPTION
**Pull request by supunTE**
Related #76 
- Design customizable race track background.

`<= 2 Players`
image

`<= 4`
image

`<=6`
image

`>6`
image

### CODE REVIEW

File: apps/client/src/pages/race/templates/race-data.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * @param count Number of players
+ * @returns Number of pavement rows
+ */
+export const getRaceTrackPavementRows = (count: number): number => {
+  return count <= 2 ? 4 : count <= 4 ? 3 : count <= 6 ? 2 : 0;
+};
+export const getRaceTrackHeight = (totalRows: number): number => {
+  return totalRows * raceTileSize * raceTrackRowColumnMultiplier;
+};
+export const getRaceTrackRowColumnSizes = (): number => {
```
Comment (dalanad) [inline apps/client/src/pages/race/templates/race-data.ts:36]: Missing comments on above three functions. also add an empty line between functions

File: apps/client/src/pages/race/Race.page.tsx [hunk 2]
```diff
@@ -1,16 +1,37 @@
...
+
+  useEffect(() => {
+    const racesModel = game.racesModel;
+
+    // TODO: Race id should return from the url
+    const raceId = Object.keys(racesModel)[0] as AppRaceId;
+    setRaceId(raceId);
+
+    return () => {
+      setRaceId(null);
```
Comment (dalanad) [inline apps/client/src/pages/race/Race.page.tsx]: Do we need to do this?

**Review by dalanad — CHANGES_REQUESTED** [review]

File: apps/client/src/pages/race/templates/race-data.ts (hunk 1 above)
Comment (supunTE) [reply apps/client/src/pages/race/templates/race-data.ts:36]: updated 3bff0754371ddd4c7c8fa3206af516590755243f

File: apps/client/src/pages/race/Race.page.tsx (hunk 2 above)
Comment (supunTE) [reply apps/client/src/pages/race/Race.page.tsx]: Removed

**Review by dalanad — APPROVED** [review]

**Pull request merged**